pkg/rpcservice/middleware.go

```diff
@@ -29,6 +48,21 @@ func authorize(ctx context.Context) error {
 	return nil
 }
 
+// Populate the context with:
+// 1. The telemetry client from the RPC Service
+// 2. The event metadata
+func updateContextWithTelemetry(ctx context.Context, methodName string, server *RPCService) context.Context {
+	// if getting the config errors, don't fail running the command
+	merchant, _ := server.cfg.UserCfg.Profile.GetAccountID()
+
+	telemetryMetadata := stripe.NewEventMetadata()
```

Comment:
could the `telemetryMetada` be empty? if its empty, should we still update the context?

Reply:
Good question! I don't think it can be since we own the `NewEventMetadata` function and we know that if will create an instance of the metadata. It's possible that not all fields in there would be set as we'd expect but i think we'd still want the metadata in the context in that case so we can still send telemetry events